src/main/jbake/content/community/docs/developer-guide/alerts-v2.adoc

```diff
@@ -872,6 +872,253 @@ Url can be a list of valid link:https://hc.apache.org/httpcomponents-core-ga/htt
 | Used for X-Proxy-Remote-User HTTP header.
 |=======================
 
+== Prometheus Integration
+
+=== Prometheus Alerter
+
+The Prometheus Alerter will listen for triggers tagged with "prometheus" tag. the Prometheus evaluations and interacts with the Alerts system. +
+ +
+The Prometheus Alerter uses the following conventions for trigger tags and context:
+
+[cols="^2,10"]
+|=======================
+| *Required* |
+_trigger.tags["prometheus"] // the value is ignored_ +
+ +
+| *Optional* |
+_trigger.context["prometheus.frequency"] = "<seconds between queries to Prometheus, default = 120>"_ +
+ +
+Note that the same frequency will apply to all Prometheus external conditions on the trigger
+| *Optional* |
+_trigger.context["prometheus.url"] = "<url, default = global setting>"_
+| *Required* |
+An ExternalCondition must be defined to be processed by the Prometheus External Alerter:
+ +
+externalcondition.alerterId = "prometheus" +
+externalcondition.expression = <BooleanExpression> | <ALERTSExpression> +
+|=======================
+
+=== Prometheus Alerter Endpoints
+
+|===
+| GET | /hawkular/alerter/status | link:#++GET_P8S_STATUS++[Status endpoint for Prometheus Alerter Integrations]
+| POST | /hawkular/alerter/notifications | link:#++POST_P8S_NOTIFICATIONS++[Listen Prometheus Notifications.]
+|===
+
+==============================================
+
+[[GET_P8S_STATUS]]
+*Endpoint GET `/awkular/alerter/status`*
+
+Ping status of the Prometheus Alerter +
+
+*Response*
+
+*Status codes*
+[cols="^20,55,^25", options="header"]
+|=======================
+|Status Code|Reason|Response Model
+
+|200|Prometheus Alerter is up and running|Success
+
+|=======================
+
+==============================================
+
+
+==============================================
+
+[[POST_P8S_NOTIFICATIONS]]
+*Endpoint POST `/awkular/alerter/notifications`*
+
+Receive a Prometheus Notification and store it as a Hawkular Event. Expects a format like: +
+
+```
+{
+  "receiver":"...",
+  "status":"...",
+  "alerts":[̣
+    {
+      "status":"...",
+      "labels":{},
+      "annotations":{},
+      "startsAt":"...",
+      "endsAt":"...",
+      "generatorURL":"..."
+    },
+    {...},
+    {...}
+  ],
+  "groupLabels":{},
+  "commonAnnotations":{},
+  "externalUrl":"...",
+  "version":"...",
+  "groupKey":"..."
+}
+```
+
+A Prometheus Notification is mapped into a Hawkular Event with the following convention: +
+ +
+
+[cols="15,15", options="header"]
+|=======================
+|Prometheus Notification|Hawkular Event
+
+|notification.groupLabels.tenant|event.tenantId
+|notification.groupLabels.dataId|event.dataId
+|"prometheus"|event.category
+|notification.commonAnnotations.description|event.text
+|notification.commonAnnotations.summary|event.text
+|notification.groupLabels.alertName|event.text
+|notification.groupLabels.alertName|event.context.alertName
+|notification|event.context.json
+
```

Comment:
We may need to be more clear about how the event.text field is set.  It is the first non-empty value of:
1) notification.commonAnnotations.description
2) notification.commonAnnotations.summary
3) notification.groupLabels.alertName
4) "notification"